css-positioning/absolute.css

```diff
@@ -0,0 +1,38 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
```

Comment:
the relative positioning should be set here to control the absolutely positioned elements.